lib/parser/objects.ts

```diff
@@ -104,7 +104,7 @@ export class OIRange implements Range {
   }
   getLimitedRange(limit = 5) {
     const newEnd = this.end.line - this.start.line > limit ?
-      new OI(this.parent, this.start.line + limit, Number.POSITIVE_INFINITY)
+      new OI(this.parent, this.start.line + limit, 10000)
```

Comment:
I would much rather catch this in OI?

Reply:
```ts
if (j === Number.POSITIVE_INFINITY) {
        const lines = (this.parent instanceof OFile ? this.parent : this.parent.rootFile).text.slice(0, this.i).split('\n');
        j = lines[i].length - 1;
      }
```
To the constructor of OI?
With this this gets caught everywhere?
(I did not test this ;))